internal/flink/command_artifact.go

```diff
@@ -0,0 +1,63 @@
+package flink
+
+import (
+	"github.com/spf13/cobra"
+
+	connectcustompluginv1 "github.com/confluentinc/ccloud-sdk-go-v2/connect-custom-plugin/v1"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+type customPluginSerializedOut struct {
+	Name           string `serialized:"name"`
+	Id             string `serialized:"id"`
+	ConnectorClass string `serialized:"connector_class"`
+	ContentFormat  string `serialized:"connector_class"`
+	Scope          string
```

Comment:
Why isn't this serialized?

Reply:
`scope` is not a part of response payload for custom connector api. but we will need this info for flink case. hense we populate it for EA only

Reply:
> scope is not a part of response payload for custom connector api

Why not?

Reply:
flink udf is reusing exsisting custom connector plugin api for EA users.